### DISCUSSION TITLE
detailed user documentation

### DISCUSSION THREAD

**Discussion by stephanfriedrich** [dani-garcia/vaultwarden#4712] [Q&A]
hello, 
thanks for vaultwarden.  

i would like to use vaultwarden in the community and need a simple, user-friendly documentation on CRUD in user management, password/vault management and the use of different clients.  

since i couldn't really find anything comprehensive, i wanted to ask here.  

best regards

**Comment by BlackDex** [comment]
Check https://bitwarden.com/help/ since all the clients are from Bitwarden and not this project.
